app/main.js

```diff
@@ -1 +1,8 @@
 chrome.power.requestKeepAwake("display");
+
+window.addEventListener('keydown', event => {
+  // F5
+  if (event.keyCode == 116) {
+    webview.reload();
```

Comment:
Could we make this restart the whole app instead (to pick up new versions of the app)?

I think it's `chrome.runtime.reload();`